providers/logr/examples_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 	"time"
 
-	grpclogr "github.com/grpc-ecosystem/go-grpc-middleware/providers/logr/v2"
+	grpclogr "github.com/grpc-ecosystem/go-grpc-middleware/providers/logr"
```

Comment:
OK this is now becoming a pattern so; what is the rule for the third import group? I might suggest we just change it all to 2 import grouping instead of 3 to avoid this issue.

Reply:
Yea, simpler is better - will adjust.